src/components/Header/Header.jsx

```diff
@@ -90,9 +90,13 @@ export function Header() {
 					</div>
 					<div className='flex items-center gap-x-5 md:gap-x-8'>
 						<Button href='/signup' color='blue'>
-							<span>
-								Sign up <span className='hidden lg:inline'>today</span>
-							</span>
+							{session ? (
+								<span>Submit a question</span>
```

Comment:
The *submit a question* button should lead people to the question submission page (I believe it's at /questions/new).

Reply:
Missed it, working on it.